src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1029,46 +1029,46 @@ else if (fieldClass == Pattern[].class) {
             result = "Regular Expressions";
         }
         else if (fieldClass == SeverityLevel.class) {
-            result = "Severity";
+            result = "Severity Level";
```

Comment:
Issue says there should be no space. I was under the impression enums display name should be the same as the class name.

Reply:
Done.